zyngui/zynthian_gui_base.py

```diff
@@ -412,26 +412,4 @@ def cb_topbar(self,event):
 		self.zyngui.zynswitch_defered('S',1)
 
 
-	def cb_keybinding(self, event):
```

Comment:
I see you moved this to zynthian_gui and captured events from the top window.... this is completely OK for me ;-)

Reply:
Yes - we want to have consistent key capture across the application and putting this at the top level allows both a more consistent implementation and more uniform access to the binding.